File: mod_02_logica_e_algoritmos/imc/java_imc_oo/src/Imc.java
abstract class Imc {
    protected double peso;
    protected double altura;

    public Imc(double peso, double altura) {
        this.peso = peso;
        this.altura = altura;
    }

    public double getImc() {
        return this.peso / Math.pow(this.altura, 2);
    }

    public abstract String classificacao();
}

File: mod_02_logica_e_algoritmos/imc/python_imc_oo/Imc.py
from abc import ABC, abstractmethod

class Imc(ABC):
    def __init__(self, peso, altura):
        self.peso = peso
        self.altura = altura

    def getImc(self):
        return self.peso / pow(self.altura, 2)

    @abstractmethod
    def classificacao(self):
        pass
